api/sports-grid/sports-grid.js

```diff
@@ -3,10 +3,16 @@ const router = express.Router()
 
 const basketballData = require('../game_data/basketBallTeamData.json');
 const { constructPuzzle } = require('./construct-puzzle');
+const { getPlayerList } = require('./get-player-list');
 
 router.get('/basketball', (_, res) => {
     const puzzle = constructPuzzle(basketballData);
     res.json(puzzle);
 })
 
+router.get('/basketball/players', (_, res) => {
+    const players = getPlayerList();
+    res.json(players);
+});
+
```

Comment:
i wonder if it's better to include these in one api call. will we ever need to have separate calls?